src/main/java/org/takes/facets/auth/social/PsTwitter.java

```diff
@@ -65,26 +72,60 @@
     private final transient String key;
 
     /**
+     * Request for fetching app token.
+     */
+    private final transient com.jcabi.http.Request trequest;
+
+    /**
+     * Request for verifying user credentials.
+     */
+    private final transient com.jcabi.http.Request vcrequest;
+
+    /**
      * Ctor.
-     * @param gapp Twitter app
-     * @param gkey Twitter key
+     * @param tapp Twitter app
+     * @param tkey Twitter key
+     */
+    public PsTwitter(final String tapp, final String tkey) {
```

Comment:
I cannot be done, because:
[ERROR] \src\main\java\org\takes\facets\auth\social\PsTwitter.java[89]: 'app' hides a field. (HiddenFieldCheck)
[ERROR] \src\main\java\org\takes\facets\auth\social\PsTwitter.java[89]: 'key' hides a field. (HiddenFieldCheck)